core/chains/evm/txmgr/nonce_tracker_test.go

```diff
@@ -61,8 +61,8 @@ func TestNonceTracker_LoadSequenceMap(t *testing.T) {
 
 		randNonce1 := testutils.NewRandomPositiveInt64()
 		randNonce2 := testutils.NewRandomPositiveInt64()
-		client.On("PendingNonceAt", mock.Anything, addr1).Return(uint64(randNonce1), nil).Once()
-		client.On("PendingNonceAt", mock.Anything, addr2).Return(uint64(randNonce2), nil).Once()
+		client.On("NonceAt", mock.Anything, addr1, (*big.Int)(nil)).Return(uint64(randNonce1), nil).Once() //nolint:gosec // Disable G115: randNonce1 always positive
```

Comment:
Think in all of the other mocks we're using `mock.Anything`. Any reason we're using `(*big.Int)(nil)` here instead?

Reply:
right! No particular reason. I'll change these to mock.Anything for consistency